src/screens/Home/components/AboutUs/i18n.ts

```diff
@@ -0,0 +1,22 @@
+import i18next from 'i18next';
+
+i18next.addResources('es', 'Comments', {
+  title: 'QUÉ DICEN NUESTROS SOCIOS',
+  one: 'Lorem Ipsum is simply dummy text of the printing and\n typesetting industry. Lorem Ipsum has been the industrys standard\n dummy text ever since the 1500s, when an unknown print',
+  nameOne: ' -John Doe',
+  two: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, \nsed do eiusmod tempor incididunt ut labore et dolore magna aliqua.\n Ut enim ad minim veniam, quis nostrud exercitation \nullamco laboris nisi ut aliquip ex ea commodo consequat. ',
+  nameTwo: ' -John Doe',
+  three:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, \nsed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+  nameThree: ' -John Doe'
+});
+
+i18next.addResources('en', 'Comments', {
+  title: 'What Our Partners Say',
+  one: 'text. ',
+  nameOne: '-John Doe',
+  two: 'text ',
+  nameTwo: '-John Doe',
+  three: 'text.',
+  nameThree: '-John Doe'
+});
```

Comment:
Put either the same lorem ipsum text or some other stuff to fill this thing correctly.

Also, the `-` on the `John Doe`s has to be on the component itself, not on the constant. So you can only fill the name in here without the dash

Reply:
done